ansible/roles/fanout/templates/sonic_deploy_cisco_8101_202205.j2

```diff
@@ -0,0 +1,715 @@
+        "192.0.0.21": {},
+        "192.0.0.22": {},
+        "192.0.0.23": {},
+        "192.0.0.24": {},
+        "192.0.0.25": {},
+        "192.0.0.26": {},
+        "192.0.0.27": {},
+        "192.0.0.28": {},
+        "192.0.0.29": {},
+        "192.0.0.30": {},
+        "192.0.0.31": {},
+        "192.0.0.32": {},
+        "192.0.0.33": {},
+        "192.0.0.34": {},
+        "192.0.0.35": {},
+        "192.0.0.36": {},
+        "192.0.0.37": {},
+        "192.0.0.38": {},
+        "192.0.0.39": {},
+        "192.0.0.40": {},
+        "192.0.0.41": {},
+        "192.0.0.42": {},
+        "192.0.0.43": {},
+        "192.0.0.44": {},
+        "192.0.0.45": {},
+        "192.0.0.46": {},
+        "192.0.0.47": {},
+        "192.0.0.48": {}
+    },
+    "DSCP_TO_TC_MAP": {
+        "AZURE": {
+            "0": "1",
+            "1": "1",
+            "10": "1",
+            "11": "1",
+            "12": "1",
+            "13": "1",
+            "14": "1",
+            "15": "1",
+            "16": "1",
+            "17": "1",
+            "18": "1",
+            "19": "1",
+            "2": "1",
+            "20": "1",
+            "21": "1",
+            "22": "1",
+            "23": "1",
+            "24": "1",
+            "25": "1",
+            "26": "1",
+            "27": "1",
+            "28": "1",
+            "29": "1",
+            "3": "3",
+            "30": "1",
+            "31": "1",
+            "32": "1",
+            "33": "1",
+            "34": "1",
+            "35": "1",
+            "36": "1",
+            "37": "1",
+            "38": "1",
+            "39": "1",
+            "4": "4",
+            "40": "1",
+            "41": "1",
+            "42": "1",
+            "43": "1",
+            "44": "1",
+            "45": "1",
+            "46": "5",
+            "47": "1",
+            "48": "6",
+            "49": "1",
+            "5": "2",
+            "50": "1",
+            "51": "1",
+            "52": "1",
+            "53": "1",
+            "54": "1",
+            "55": "1",
+            "56": "1",
+            "57": "1",
+            "58": "1",
+            "59": "1",
+            "6": "1",
+            "60": "1",
+            "61": "1",
+            "62": "1",
+            "63": "1",
+            "7": "1",
+            "8": "0",
+            "9": "1"
+        }
+    },
+    "FEATURE": {
+        "acms": {
+            "auto_restart": "enabled",
+            "has_global_scope": "True",
+            "has_per_asic_scope": "False",
+            "has_timer": "False",
+            "high_mem_alert": "disabled",
+            "set_owner": "local",
+            "state": "enabled"
+        },
+        "bgp": {
+            "auto_restart": "enabled",
+            "check_up_status": "false",
+            "has_global_scope": "False",
+            "has_per_asic_scope": "True",
+            "has_timer": "False",
+            "high_mem_alert": "disabled",
+            "set_owner": "local",
+            "state": "enabled"
```

Comment:
Do we need these feature enabled?

Can you check acms/bgp/dhcp_relay/telemetry/... ?
You can set default disabled if they are no need for fanout

Reply:
disabled

Reply:
I can't see bgp/teamd/... disabled. Need to write as below:
```
"state": "disabled"
```

Reply:
@Xichen96 You should disable feature in `FEATURE` table. e.g., to disable bgp, update line 412, not line 134.